src/session/components/avatar.rs

```diff
@@ -127,22 +128,33 @@ impl Avatar {
         let imp = self.imp();
 
         // Chat title expression
-        let title_expression = item_expression.chain_property::<Chat>("title");
+        let is_deleted_expression = expressions::is_deleted_expression(&item_expression);
+        let title_expression = item_expression.chain_property::<Chat>("title").upcast();
```

Comment:
Here you can just use `expressions::chat_display_name`. This way you don't need the manual handling with the `is_deleted_expression`.